metriken-exposition/src/parquet.rs

```diff
@@ -179,9 +185,24 @@ impl ParquetSchema {
             }
         }
 
+        let metadata: Option<Vec<KeyValue>> = if self.metadata.is_empty() {
+            None
+        } else {
+            Some(
+                self.metadata
+                    .drain()
```

Comment:
`into_iter` is a better match for what you are trying to do here
```suggestion
                self.metadata
                    .into_iter()
```